eternalcore/src/main/java/com/eternalcode/core/EternalCore.java

```diff
@@ -268,14 +269,15 @@ public void onEnable() {
         /* Services & Managers  */
 
         this.userManager = new UserManager();
-        this.placeholderRegistry = new PlaceholderRegistry();
-        this.placeholderRegistry.stack(text -> {
+        this.placeholderRegistry = new PlaceholderBukkitRegistryImpl(server);
+        this.placeholderRegistry.registerPlaceholderReplacer(text -> {
             for (Map.Entry<String, String> entry : this.placeholdersConfiguration.placeholders.entrySet()) {
                 text = text.replace(entry.getKey(), entry.getValue());
             }
 
             return text;
         });
+        this.placeholderRegistry.registerPlayerPlaceholderReplacer(new PlaceholderReplacerApiReplacer());
```

Comment:
Zrobić hookManagera, który sprawdza czy jest papi na serwerze jeżeli tak to rejestruje replacer